voxpupuli-acceptance.gemspec

```diff
@@ -29,7 +29,6 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'puppet-modulebuilder', '~> 0.1'
   s.add_runtime_dependency 'rake'
   s.add_runtime_dependency 'rbnacl', '>= 4'
```

Comment:
It was added in https://github.com/voxpupuli/modulesync_config/pull/478 so net-ssh could use ed25519 keys but it looks like since net-ssh 5.0 rbnacl is no longer needed. Should we drop `rbnacl` altogether now?
```suggestion
```

Reply:
@genebean what do you think about this? IMO we should depend on net-ssh >= 5. We already do this in https://github.com/voxpupuli/beaker/blob/master/beaker.gemspec#L41 . That allows us to drop rbnacl/rbnacl-libsodium dependencies. Also IMO the ed2519 gem should probably be added to beaker as dependency, not voxpupuli-acceptance?

Reply:
That all makes sense to me.